draft-regext-rdap-extensions.md

```diff
@@ -797,17 +833,15 @@ within the same namespace of an existing RDAP extension without changing the
 extension identifier or other signaling methods (see [@?I-D.ietf-regext-rdap-versioning]).
 
 In this scenario, clients that are not updated to recognize the new elements
-should simply ignore them. The same is also true for referrals (see (#referrals)).
-
-However, the introduction of new object classes into an existing extension will
-cause most clients to process no information and will cause some clients to produce
-errors.
-
-Extensions MUST NOT be evolved as described in this section because there is no
-explicit signal to clients regarding these extensions. This lack of signal
-will lead to difficulty in troubleshooting issues and could mislead client
-implementers to believe their software is fully conforming with the extension
-specification when it is not.
+should simply ignore them. This is true for all non-breaking changes
+(see (#nonbreaking_changes)).
+
+This lack of signaling mechanism can lead to difficulty in troubleshooting
```

Comment:
"This lack ..." -> "The lack ..." ? Since "This" seems to connote a prior context but couldn't perceive it.